webrtc.html

```diff
@@ -8068,7 +8063,10 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
                   <code><a data-for="RTCDataChannel">negotiated</a></code>,
                   <code><a data-for="RTCDataChannel">id</a></code> and
                   <code><a data-for="RTCDataChannel">priority</a></code> attributes
-                  to the values of the corresponding members of <var>init</var>.
+                  to the values of the corresponding members of the
+                  <var>dataChannelDict</var> argument, using a value of
+                  <code>null</code> if the corresponding dictionary member is
```

Comment:
A note here spelling out which 3 members can be missing would be good. Syntactically `{}` is missing all of them, but only those without a default value can be missing here, after the conversion to an IDL dictionary.